db/models/order.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const {STRING, INTEGER, DOUBLE} = require('sequelize')
+
+module.exports = db => db.define('orders', {
+  orderNumber: STRING,
+  status: STRING,
+  qty: INTEGER,
+  price: DOUBLE
+}, {
+  getterMethods: {
+    getOrdNum: () =>
+      `SS-${new Date().toISOString().slice(0, 10)}-${Math.floor(Math.random()*100)+400}`
+  },
+  instanceMethods: {
+    addItem: function(item) {
+      item.orderNumber = this.orderNumber
+      return db.Order.create(item)
```

Comment:
db.Order.create is creating a new order, not adding an item to an existing order. I would create an association Product.belongsToMany(Order), and then you will have the methods that Sequelize gives you to get all the products associated with a order.